src/test/java/org/gridsuite/loadflow/server/LoadFlowParametersTest.java

```diff
@@ -0,0 +1,207 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.loadflow.server;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.gridsuite.loadflow.utils.assertions.Assertions.*;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import org.gridsuite.loadflow.server.dto.parameters.LoadFlowParametersInfos;
+import org.gridsuite.loadflow.server.entities.parameters.LoadFlowParametersEntity;
+import org.gridsuite.loadflow.server.repositories.parameters.LoadFlowParametersRepository;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
```

Comment:
Start with JUnit5 and not obsolete JUnit4
```suggestion
import org.junit.jupyter.api.AfterEach;
import org.junit.jupyter.api.Before;
import org.junit.jupyter.api.Test;
```